app/views/inventory/viewStockLevels.html

```diff
@@ -0,0 +1,52 @@
+<div class="col-sm-3 col-md-2" ng-include="'views/inventory/sidebar.html'"></div>
+<div class="col-sm-9 col-md-10 main">
+  <h3 class="page-header">View Stock Levels</h3>
+
+  <hr />
+
+  <form name="searchForm" class="form-horizontal col-sm-12" style="padding:10px 0;" ng-submit="viewStockLevels(searchForm)" novalidate>
+
+    <div class="clearfix form-group" ng-class="{'has-error': searchForm.allSites.$invalid}">
+        <label class="col-sm-2 control-label">Distribution Center:</label>
+        <div class="form-inline row col-sm-10">
+          <select name="distributionCenter" class="form-control" ng-model="search.distributionCenter" ng-change="updateAllSites()" style="min-width: 350px;">
+            <option value="" disabled></option>
+            <option ng-repeat="distributionCenter in distributionCenters" value="{{distributionCenter.id}}">
+              {{distributionCenter.name}}
+            </option>
+          </select>
+          <input type="checkbox" name="allSites" ng-model="search.allSites" ng-click="clearDistributionCenter()" ng-required="!search.distributionCenter" />
+          <label>All Venues</label>
+          <ng-messages for="searchForm.allSites.$error" role="alert" ng-show="searchForm.allSites.$dirty">
+              <ng-messages-include src="messages.html"></ng-messages-include>
+          </ng-messages>
+        </div>
+    </div>
+
+    <div class="col-sm-offset-2">
+      <input type="radio" name="inventoryStatus" ng-model="search.inventoryStatus" value="IN_STOCK" required />
+      <label>In Stock</label>
+      <input type="radio" name="inventoryStatus" ng-model="search.inventoryStatus" value="NOT_LABELLED" required />
+      <label>Not Labelled</label>
```

Comment:
@timler Same as above for these radio buttons https://getbootstrap.com/css/#checkboxes-and-radios.